### DISCUSSION TITLE
Problem with Documentation and Missing Components & functions

### DISCUSSION THREAD

**Discussion by FarhanMS123** [microsoft/fluentui#26103] [Q&A] [answered]
Hi! I'm bit interest with Fluent UI 9 and I want to try it. But After I read the documentation from [here](https://react.fluentui.dev/). There is a lot of documentation missing. For example, this one:

![image](https://user-images.githubusercontent.com/16165479/209508045-eab2a013-2a28-4e91-82f0-97a7db06b77d.png)

I want to use `useMenuListContainerStyles` hooks. But my ESLint cannot find any function named it. I read through documents and even v8, also installing v8, and I can't find it.

Second, I want to use some components from the web, those are Tree View or Sidebar and Indicator

![image](https://user-images.githubusercontent.com/16165479/209508250-11a37c63-fdd7-48a9-85c2-b3b40a7b0f9d.png)
![image](https://user-images.githubusercontent.com/16165479/209508378-905df7bf-eb8e-4e82-b1ce-1388659a9d41.png)

What my thought is... If the documentation web of styles guidelines used some component, there must also be exists in the library, right?

Well... how can I use these components and hooks I mention earlier? Is it just custom element so I must modify it from related component? or is it just for internal use? is there api/technical documentation which I can read?

Thanks a lot!

**Comment by layershifter** [comment, marked as answer]
Hey, it's a problem with our Storybook #20367. You can check the folder with stories ([/packages/react-components/react-menu/stories](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-menu/stories)), it contains all stories with complete code examples.

**Reply by FarhanMS123** [reply]
Thankyou for the information.
